db/config.php

```diff
@@ -0,0 +1,15 @@
+<?php
+$servername = "localhost";
+$username = "myuser";
+$password = "mypassword";
+try {
+    $conn = new PDO("mysql:host=$servername;dbname=mydb", $username, $password);
```

Comment:
Do this in `get_info.php`. Have `config.php` set the `$servername`, `$username`, and `$password` variables only.